sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -62,7 +62,7 @@ class OpenTelemetrySpan(HttpSpanMixin, object):
     :paramtype context: Dict[str, str]
     """
 
-    def __init__(self, span: Optional[Span] = None, name: str = "span", **kwargs: Any) -> None:
+    def __init__(self, span: Optional[Span] = None, name: Optional[str] = "span", **kwargs: Any) -> None:
```

Comment:
Did you add Optional here because `None` is valid to pass?

Reply:
Correct, this makes it match the protocol.